backend/inspirehep/assign/utils.py

```diff
@@ -41,27 +43,68 @@ def update_author_bai(to_author_bai, lit_author):
     return lit_author_ids_list_updated
 
 
-def can_claim(data, author_profile_recid):
-    current_user_orcid = get_current_user_orcid()
-    try:
-        current_author_profile = AuthorsRecord.get_record_by_pid_value(
-            current_user_orcid, "orcid"
-        )
-    except PIDDoesNotExistError:
-        return False
-
+def _get_lit_record_from_db(control_number):
     try:
         lit_record = (
             db.session.query(RecordMetadata, PersistentIdentifier)
             .with_entities(RecordMetadata.json)
             .filter(
-                PersistentIdentifier.pid_value == str(data["control_number"]),
+                PersistentIdentifier.pid_value == str(control_number),
                 PersistentIdentifier.pid_type == "lit",
                 RecordMetadata.id == PersistentIdentifier.object_uuid,
             )
             .one()
         )
+        return lit_record[0]
```

Comment:
why we even have a tuple here? `.one()` should be enough

Reply:
when you query with entities `one` returns a tuple?